src/Microsoft.ML.FastTree/FastTree.cs

```diff
@@ -82,6 +82,31 @@ public abstract class FastTreeTrainerBase<TArgs, TPredictor> :
 
         protected string InnerArgs => CmdParser.GetSettings(Host, Args, new TArgs());
 
+        internal const string Remarks = @"<remarks>
+<para>FastTrees is an implementation of FastRank. FastRank is an efficient implementation of the <a href='https://arxiv.org/abs/1505.01866'>MART</a> gradient boosting algorithm. 
```

Comment:
"FastTrees is" -> "FastTree is"... Also they will have no idea what FastRank is. Bringing in FastRank just complicates the picture a bit. The long history of Microsoft's internal tools, while possibly interesting to some people, does not belong in the first few sentences of user document.